### ISSUE TITLE
Assertion failed: head->cf->redis.storage_mode >= REDIS_MODE_DISTRIBUTED (/usr/src/nchan-1.2.4/src/store/memory/memstore.c: memstore_fakesub_add: 738)

### ISSUE THREAD

**Issue by solofeed** [slact/nchan#514]
Nginx: 1.15.6
Nchan: 1.2.3
or
Nginx: 1.15.9
Nchan: 1.2.4

This problem produces when to adding nchan_redis_pass for multi-sub
```
location ~ /multisub-split/(.*)$ {
        nchan_authorize_request /auth;
        nchan_subscriber;
        nchan_redis_pass redis_server;
        nchan_channel_id "$1";
        nchan_channel_id_split_delimiter ",";
      }
```

For single sub all is ok
```
location ~ /sub/(\w+)$ {
        nchan_authorize_request /auth;
        nchan_subscriber;
        nchan_redis_pass redis_server;
        nchan_channel_id $1;
    }
```

**Comment by solofeed** [comment]
Here is nchan settings

```
nchan_max_reserved_memory 128M;
nchan_message_buffer_length 100;
nchan_subscriber_timeout 86400;
nchan_redis_namespace nchan;
nchan_redis_storage_mode backup;

upstream redis_server {
    nchan_redis_server "redis://redis:6379/2";
}
```

**Comment by slact** [comment]
fixed in cd602a74

**Comment by solofeed** [comment]
@slact thank you, could you please tag this fix?

**Comment by slact** [comment]
this fix is now included in the 1.2.5 release. Thanks for the bug report!
